syft/federated/train_config.py

```diff
@@ -4,9 +4,9 @@
 import torch
 
 import syft as sy
-from syft import workers
-from syft.generic import pointers
-from syft.workers import AbstractWorker
+from syft.generic.pointers.object_wrapper import ObjectWrapper
```

Comment:
I would prefer to follow the google style guide for import statements: https://github.com/google/styleguide/blob/gh-pages/pyguide.md, saying (section 2.2):

> Use import statements for packages and modules only, not for individual classes or functions. Note that there is an explicit exemption for imports from the typing module.

The advantage is that you always know, from which module your classes are coming from.

Reply:
I'm not sure how I feel about it, but I suggest moving this discussion into an issue or the changes into a separate PR. This one is already quite large, and it should be easier to do once imports are standardized after this merges.

Reply:
It is definitely a discussion that is out of the scope of this pull request. However you are also changing the import statements that fulfilled the google style guide. So we cannot just postpone the discussion.

Reply:
There were very few existing import statements that fulfilled the requirement you quoted, and the existing import statements are clearly not yet standardized, so I don't think that should be a blocker to this PR. I will make an issue and link this thread to it.